upload-riffraff.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
I'm a bit confused by this file. I don't understand why the upload is separate from the normal `continuous-integrations.sh` and I'd expect the ivy/lock cleanup stuff in the `.travis.yml`, like with https://github.com/guardian/status-app/blob/travis-riffraff-artifact/.travis.yml